### PR TITLE
Add(route): add 广州天气预警

### LINKED ISSUE #1882: 广州天气预警
### 源站地址
http://www.tqyb.com.cn/
### 需要生成什么内容？
- [1] 突发性天气提示（http://www.tqyb.com.cn/gz/weatherAlarm/suddenWeather/）
- [2] 广东省内城市预警信号（http://www.tqyb.com.cn/gz/weatherAlarm/otherCity/）

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #1882

## 完整路由地址 / Example for the proposed route(s)

```routes
/gov/guangdong/tqyb/tfxtq
/gov/guangdong/tqyb/sncsyjxh
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [X] New Route
- [X] Documentation
  - [X] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] 日期和时间 date and time
  - [X] 可以解析 Parsed
  - [X] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/government.md [hunk 1]
```diff
@@ -90,6 +90,16 @@ pageClass: routes
...
+## 广州天气
+
+### 突发性天气提示
+
+<Route author="Fatpandac" example="/gov/guangdong/tqyb/tfxtq" path="/gov/guangdong/tqyb/tfxtq"/>
+
+### 广东省内城市预警信号
+
+<Route author="Fatpandac" example="/gov/guangdong/tqyb/sncsyjxh" path="/gov/guangdong/tqyb/sncsyjxh"/>
+
```
Comment (TonyRL) [inline docs/government.md:102]: Please move it to forecast.md

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/government.md (hunk 1 above)
Comment (Fatpandac) [reply docs/government.md]: Done.

**Comment by TonyRL** [comment]
pubDate on `/gov/guangdong/tqyb/sncsyjxh` is wrong. You can compare it with the description.

**Comment by Fatpandac** [comment]
> pubDate on `/gov/guangdong/tqyb/sncsyjxh` is wrong. You can compare it with the description from github-actions.

The pubDate of `sncsyjxh` is right. I can't find out where is wrong.
image

**Comment by TonyRL** [comment]
> > pubDate on `/gov/guangdong/tqyb/sncsyjxh` is wrong. You can compare it with the description from github-actions.
> 
> The pubDate of `sncsyjxh` is right. I can't find out where is wrong. image

11:53:00 **GMT** vs 2022年01月19日 11:53

**Comment by Fatpandac** [comment]
@TonyRL Yeh, you are right.

**Pull request merged**